android/beagle/src/main/java/br/com/zup/beagle/android/components/Button.kt

```diff
@@ -78,7 +78,10 @@ data class Button(
             preFetchHelper.handlePreFetch(rootView, it)
         }
 
-        val button = viewFactory.makeButton(rootView.getContext(), styleManager.getButtonStyle(styleId))
+        val style = styleManager.getButtonStyle(styleId)
+
+        val button = if (style == 0) viewFactory.makeButton(rootView.getContext())
+        else viewFactory.makeButton(rootView.getContext(), style)
```

Comment:
What do you think about putting `0` in a constant property to make maintenance more intuitive? For example `NO_STYLE`.
```suggestion
        val button = if (style == NO_STYLE) {
            viewFactory.makeButton(rootView.getContext())
        } else {
            viewFactory.makeButton(rootView.getContext(), style)
        }
```
Personally, I think that adding `{}` to the if else and replacing `0` with a constant has little relevance, feel free to decide.